src/main/java/com/swervedrivespecialties/exampleswerve/commands/ToggleMinSpeedScale.java

```diff
@@ -0,0 +1,44 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.swervedrivespecialties.exampleswerve.commands;
+
+import com.swervedrivespecialties.exampleswerve.subsystems.DrivetrainSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ToggleMinSpeedScale extends CommandBase {
+  
+  private DrivetrainSubsystem _drive;
+
+  public ToggleMinSpeedScale(DrivetrainSubsystem drive) {
```

Comment:
Same deal as before here. Whether or not you pass in subsystems is something unclear enough that you should do with it what you feel best. But I recommend you just adopt a strong singleton pattern.